src/components/VAllResultsGrid/VAllResultsGrid.vue

```diff
@@ -4,14 +4,14 @@
       v-if="!noResults"
       class="results-grid grid grid-cols-2 lg:grid-cols-5 2xl:grid-cols-6 gap-4 mb-4"
     >
-      <NuxtLink
+      <VContentLink
         v-for="[key, item] in results"
         :key="key"
-        :to="localePath({ path: `/search/${key}`, query: $route.query })"
-        class="lg:col-span-2 focus:bg-white focus:border-tx focus:ring focus:ring-pink focus:outline-none focus:shadow-ring focus:text-black rounded-sm"
-      >
-        <VContentLink :results-count="item.count" :media-type="key" />
-      </NuxtLink>
+        :media-type="key"
+        :results-count="item.count"
+        :to="{ path: `/search/${key}`, query: $route.query }"
```

Comment:
```suggestion
        :to="{ path: `/search/${key}`, query: $route.query }"
```

This should be restored to use `localePath`, try visiting `http://localhost:8443/es/search/?q=dog` and observe the missing `/es` path on these routes.

Reply:
Right! I don't know how that slipped out, thank you!